docs/modules/ROOT/pages/user-guide/bootstrap.adoc

```diff
@@ -0,0 +1,35 @@
+= Bootstrap Camel Quarkus
+:page-aliases: bootstrap.adoc
```

Comment:
```suggestion
```
If bootstrap.adoc is  a new page, there is no point in adding an alias. Aliases serve for setting redirects and we use them mainly when we move a page.